app/services/CheckoutService.scala

```diff
@@ -60,6 +62,8 @@ class CheckoutService(identityService: IdentityService,
     val plan = RatePlan(subscriptionData.productRatePlanId.get, None)
     def emailError: SubNel[Unit] = EitherT(Future.successful(\/.left(NonEmptyList(CheckoutIdentityFailure("Email in use")))))
     val idMinimalUser = authenticatedUserOpt.map(_.user)
+    val soldToOverride = subscriptionData.productData.left.toOption.filter(isGuardianWeekly).map(x => x.deliveryAddress)
```

Comment:
I'm not keen on calling things Override, but I'm not sure if we can do a getOrElse at this stage